build.sbt

```diff
@@ -6,6 +6,8 @@ val libName = "play-frontend-govuk-examples"
 
 lazy val playDir = "play-2"
 
+ThisBuild / scalaVersion := "2.13.18"
```

Comment:
we have a scala version set in the settings(...) of the root project below too

Reply:
@oscarduignan Removed, thank you - added it whilst upgrading but you're right, wasn't necessary.

Reply:
should we bump the scalaVersion in the settings bit to 2.13.18?

Reply:
or should it be 2.13.12 (from the pr-commenter)